lib/active_merchant/billing/gateways/pay_junction_v2.rb

```diff
@@ -110,6 +112,27 @@ def add_payment_method(post, payment_method)
         end
       end
 
+      def add_address(post, options)
+        if (address = options[:billing_address])
+          post[:billing] = {}
+          post[:billing][:firstName] = address[:first_name] if address[:first_name]
+          post[:billing][:lastName] = address[:last_name] if address[:last_name]
+          post[:billing][:companyName] = address[:company] if address[:company]
+          post[:billing][:phone] = address[:phone_number] if address[:phone_number]
+
+          post[:billing][:address] = {}
+          post[:billing][:address][:address] = address[:address1] if address[:address1]
+          post[:billing][:address][:city] = address[:city] if address[:city]
+          post[:billing][:address][:state] = address[:state] if address[:state]
+          post[:billing][:address][:country] = address[:country] if address[:country]
+          post[:billing][:address][:zip] = address[:zip] if address[:zip]
+        end
+      end
+
+      def get_state(address)
```

Comment:
Not sure if you meant to use this or delete it, but it is currently not used

Reply:
Thanks! Meant to delete it!